systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaConnectUtils.java

```diff
@@ -62,10 +62,11 @@ public static void waitUntilKafkaConnectRestApiIsAvailable(String namespaceName,
         LOGGER.info("KafkaConnect API is available");
     }
 
-    public static void waitForMessagesInKafkaConnectFileSink(String namespaceName, String kafkaConnectPodName, String sinkFileName, String message) {
+    public static void waitForMessagesInKafkaConnectFileSink(String namespaceName, String kafkaConnectPodName, String sinkFileName, int sinkReceivedMsgCount) {
```

Comment:
Oh it seems that everywhere was `"99"` and not the whole message.
If that's the case, I think you can keep it as it is, thanks

Reply:
good point, yeah i changed it actual int instead of String so that there would not be need for some magic of how to expect "hello-world-99" based on the fact that I am sending 100 messages and default message format is "hello-world-<msg> or smt.